spec/models/survey_spec.rb

```diff
@@ -1,3 +1,31 @@
 require 'rails_helper'
 
-describe Survey do end
+describe Survey do
+  describe 'formatted_date' do
+    it 'returns the formatted date' do
```

Comment:
So here, its better to use let blocks as opposed to variables. So for example, something like:

`let(:date) { Time.zone.parse('2010-10-10 7:30')`
`let(:survey) { create :survey, date: date }`

This way you can use them both in the same place and not have to repeat yourself. DRY code is important.